.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Smoke Test
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-20.04
+
+    container:
+      image: hexpm/elixir:1.10.4-erlang-23.1.4-ubuntu-focal-20201008
```

Comment:
This is a different image than in the other GA workflow, right? Do they need to be different?

Reply:
I'm using the same one that the test runner runs on, the don't need to be different, but this one should be this one.